game/script-topics.rpy

```diff
@@ -1824,6 +1824,70 @@ label talk_sleeping_well:
 
     return
 
+label talk_natsukis_hairstyle:
+    if jn_affinity.get_affinity_state() >= jn_affinity.ENAMORED:
+        n "Hmm?{w=0.2} My hairstyle?"
+        n "Why do you ask,{w=0.1} [player]?{w=0.2} Looking for a stylist?"
+        n "Ehehe."
+
+    elif jn_affinity.get_affinity_state() >= jn_affinity.NORMAL:
+        n "Huh?{w=0.2} My hairstyle?"
+        n "Wait...{w=0.3} are you messing with me?{w=0.2} Was that an insult?"
+        n "You better not be teasing me,{w=0.1} [player]..."
```

Comment:
I can see where you're coming from, but this isn't stemming from anger - though I can appreciate it's hard to tell without a sprite system in place! Expression-wise, Natsuki would be a little confused, followed by pouty - I'll adjust some of the writing here anyway to make that more clear.